packages/pluggableWidgets/progress-bar-web/src/components/__tests__/__snapshots__/ProgressBar.spec.tsx.snap

```diff
@@ -2,14 +2,14 @@
 
 exports[`Progress bar has progress bar structure 1`] = `
 <div
-  className="widget-progress-bar"
+  className="widget-progress-bar progress-bar-medium"
```

Comment:
@diego-antonelli one of the requirements was to not change the structure, are these css classname changes fine?

Reply:
I think is fine, the original widget-progress-bar is being kept :)